brainio/fetch.py

```diff
@@ -5,21 +5,27 @@
 import zipfile
 
 import boto3
-import pandas as pd
-import xarray as xr
 from botocore import UNSIGNED
 from botocore.config import Config
 from six.moves.urllib.parse import urlparse
 from tqdm import tqdm
 
-from brainio import assemblies as assemblies_base
-from brainio.assemblies import coords_for_dim
-from brainio.stimuli import StimulusSet
+import brainio.assemblies as assemblies
+import brainio.stimuli as stimuli
+from brainio.stimuli import StimulusSetLoader
 from brainio.lookup import lookup_assembly, lookup_stimulus_set, sha1_hash
 
-_local_data_path = os.path.expanduser(os.getenv('BRAINIO_HOME', '~/.brainio'))
+BRAINIO_HOME = 'BRAINIO_HOME'
 
 _logger = logging.getLogger(__name__)
+_local_data_path = None
+
+
+def get_local_data_path():
+    global _local_data_path
```

Comment:
Can't we do away with the global `_local_data_path` and just call this method every time?

Reply:
This makes it easier to mock.

Reply:
ok makes sense. I would add a comment saying exactly this